src/components/pages/Supporters.vue

```diff
@@ -1,42 +1,44 @@
 <template>
 	<div class="supporters">
 		<h2 class="title is-size-2">{{ pageTitle }}</h2>
-		<supporter lang="de" v-for="( supporter, index ) in supporters"
-				:key="index"
-				v-on:supporter-opened="setSupporterId( $event )"
-				v-on:supporter-closed="setSupporterId( null )"
-				:content="supporter"
-				:visible-supporter-id="visibleSupporterId"
-				:supporter-id="index">
-		</supporter>
+		<supporter
+			lang="de"
+			v-for="(supporter, index) in supporters"
+			:key="index"
+			@supporter-opened="setSupporterId($event)"
+			@supporter-closed="setSupporterId(null)"
+			:content="supporter"
+			:visible-supporter-id="visibleSupporterId"
+			:supporter-id="index"
+		></supporter>
 	</div>
 </template>
 
+<script setup lang="ts">
+import { ref } from 'vue';
+import { Supporter as SupporterInfo } from '@src/view_models/supporters';
+
+defineProps( {
+	pageTitle: String,
+	supporters: {
+		type: Array as () => SupporterInfo[],
+	},
+} );
```

Comment:
We prefer the pattern of first having an interface called `Props` and then doing `defineProps<Props>()`

The interface can use lower-case `string` types and can use `SupporterInfo[]` directly without anonymous function